lib/tasks/uploads.rake

```diff
@@ -566,13 +567,21 @@ def sync_s3_acls(async: true, concurrency: 10)
   end
 end
 
-task "uploads:sync_s3_acls", %i[synchronous parallel] => :environment do |_, args|
+# This task is defined for backwards compatibility
+task "uploads:sync_acls", %i[synchronous parallel] => :environment do |_, args|
```

Comment:
If this is defined for backwards compatibility, why was it renamed? `_s3` is missing.

Reply:
This was a mistake. Good catch 👍